src/main/resources/static/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="Our place to get know-how all things work">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Welcome in Knowall!</title>
+
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <pre><output>
+            knowall@where-it-could-be:~$ echo $HOME 
+            Our Planet, Planet Earth
+            knowall@where-it-could-be:~$ whoami
+
+
+                                               ██╗  ██╗███╗   ██╗ ██████╗ ██╗    ██╗ █████╗ ██╗     ██╗     
+                                               ██║ ██╔╝████╗  ██║██╔═══██╗██║    ██║██╔══██╗██║     ██║     
+                                               █████╔╝ ██╔██╗ ██║██║   ██║██║ █╗ ██║███████║██║     ██║     
+                                               ██╔═██╗ ██║╚██╗██║██║   ██║██║███╗██║██╔══██║██║     ██║     
```

Comment:
Nice - I love it. I really do.

Reply:
Thanks, this is just a temporary solution. Nevertheless, I am proud of it.